### ISSUE TITLE
Postgres initdb fail

### ISSUE THREAD

**Issue by aitva** [termux/termux-packages#2333]
Hi,

I just went through Postgres installation and `initdb` fail with the following error:
```
performing post-bootstrap initialization ... 2018-04-11 15:39:25.800 CEST [2544] FATAL:  could not open extension control file "/usr/share/postgresql/extension/plpgsql.control": No such file or directory
2018-04-11 15:39:25.800 CEST [2544] STATEMENT:  CREATE EXTENSION plpgsql;
```

It seems that `/usr/share/postgresql/extension/` is missing from the installation. I checked on Debian and the folder is created by the package.

**Comment by tomty89** [comment]
```
$ dpkg -S plpgsql.control
postgresql-contrib: /data/data/com.termux/files/usr/share/postgresql/extension/plpgsql.control
$
```

**Comment by aitva** [comment]
Thank you @tomty89!

I did not know about `dpkg -S`, it is very handy. I had a look at the Debian package again using your command:
```
$ dpkg -S plpgsql.control
postgresql-10: /usr/share/postgresql/10/extension/plpgsql.control
```

In Debian the `postgresql` package let me initialize a database without installing the `postgresql-contrib`, this is what I was expecting in Termux. I can update the [wiki](https://wiki.termux.com/wiki/Postgresql) but maybe this should be mark as a bug?

**Comment by tomty89** [comment]
It is a bug. See my PR for details.

**Comment by fornwall** [comment]
This is now fixed in version `10.3-2` of the postgresql package (which merges in postgresl-contrib) by @tomty89!
